src/client/graphql/QuoteCart.graphql

```diff
@@ -0,0 +1,96 @@
+query QuoteCart($id: ID!) {
+  quoteCart(id: $id) {
+    id
+    bundle {
+      possibleVariations {
+        id
+        tag(locale: sv_SE)
```

Comment:
Wouldn't be the case of getting `locale` as an argument of `QuoteCart` query?

Reply:
haha yeah sorry, this is confusing -- I actually update this query in another PR! But maybe I should fix it here already